hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonInputFormat.java

```diff
@@ -384,6 +408,32 @@ public static ReadCommittedScope getReadCommittedScope(Configuration configurati
    */
   @Override public abstract List<InputSplit> getSplits(JobContext job) throws IOException;
 
+  List<ExtendedBlocklet> getDistributedSplit(CarbonTable table,
+      FilterResolverIntf filterResolverIntf, List<PartitionSpec> partitionNames,
+      List<Segment> validSegments, List<Segment> invalidSegments, DataMapLevel level)
```

Comment:
No need to pass level here as it is always null

Reply:
removed